packages/tools/src/eventListeners/touch/touchStartListener.ts

```diff
@@ -248,12 +272,33 @@ function _onTouchDrag(evt: TouchEvent) {
   const deltaPoints =
     currentPointsList.length === lastPointsList.length
       ? getDeltaPoints(currentPointsList, lastPointsList)
-      : getDeltaPoints(currentPointsList, currentPointsList);
+      : zeroIPoint;
 
   const deltaDistance =
     currentPointsList.length === lastPointsList.length
       ? getDeltaDistanceBetweenIPoints(currentPointsList, lastPointsList)
-      : getDeltaDistanceBetweenIPoints(currentPointsList, currentPointsList);
+      : zeroIDistance;
+
+  const totalDistance =
+    currentPointsList.length === lastPointsList.length
+      ? getDeltaDistance(currentPointsList, state.lastPointsList)
+      : zeroIDistance;
+
+  state.accumulatedDistance = {
+    page: state.accumulatedDistance.page + totalDistance.page,
+    client: state.accumulatedDistance.client + totalDistance.client,
+    canvas: state.accumulatedDistance.canvas + totalDistance.canvas,
+    world: state.accumulatedDistance.world + totalDistance.world,
+  };
+
+  const clamp = (num) => Math.min(Math.max(num, -15), 10);
```

Comment:
So if I understand correctly, it is added to make the drag more "smooth". Regardless of why, please add a comment explaining why because it is not obvious why it is being done. Someone looking at the code will likely wonder (as I did) why would a drag get its distance capped?

That said, I am still surprised this is needed.

Reply:
I've commented this out and left a note on why someone might want to clamp.